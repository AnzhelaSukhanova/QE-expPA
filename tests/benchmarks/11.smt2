(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 6))
(declare-fun z () (_ BitVec 6))
(declare-fun t () (_ BitVec 6))
(assert
 (exists ((x (_ BitVec 6))) (and 
 (or (bvult x (bvmul #b000011 (bvshl #b000001 y))) (= x (bvmul #b000011 (bvshl #b000001 y)))) 
 (or (bvult (bvshl #b000001 z) x) (= (bvshl #b000001 z) x)) 
 (or (bvult x t) (= x t)))))
