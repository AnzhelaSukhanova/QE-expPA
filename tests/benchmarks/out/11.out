(let (($e1 (bvshl #b000001 z))) (and (not (bvult (bvshl #b000011 y) $e1)) (not (bvult t $e1))))
